deployments/epinio.go

```diff
@@ -165,7 +165,7 @@ func (k Epinio) apply(ctx context.Context, c *kubernetes.Cluster, ui *termui.UI,
 				strings.Contains(err.Error(), "EOF")
 		}),
 		retry.OnRetry(func(n uint, err error) {
-			ui.Note().Msgf("retrying to create the epinio cert using cert-manager")
+			ui.Note().Msgf("Retrying creation of API cert via cert-manager (%d/10)", n)
 		}),
 		retry.Delay(5*time.Second),
 		retry.Attempts(10),
```

Comment:
refactor opportunity: make `10` a variable or even better, a constant. You can use it in the message above too.

Reply:
Right, of course. Will be done.

Reply:
Done with commits b9ad829867534bab1224e007655c7c8b61b8040b, f4ade8929dbf718b249526880f2374efbb77eb50, and c772e169b2c6779124cc7279ab13c274b76139b4.